File: recovered/src/com/lumiyaviewer/lumiya/voice/common/messages/VoiceChannelClosed.java
package com.lumiyaviewer.lumiya.voice.common.messages;

import android.os.Bundle;
import com.lumiyaviewer.lumiya.voice.common.VoicePluginMessage;
import com.lumiyaviewer.lumiya.voice.common.model.VoiceChannelInfo;
import javax.annotation.Nonnull;

/* loaded from: classes.dex */
public class VoiceChannelClosed implements VoicePluginMessage {

    @Nonnull
    public final VoiceChannelInfo channelInfo;

    public VoiceChannelClosed(Bundle bundle) {
        this.channelInfo = new VoiceChannelInfo(bundle.getBundle("channelInfo"));
    }

    public VoiceChannelClosed(@Nonnull VoiceChannelInfo voiceChannelInfo) {
        this.channelInfo = voiceChannelInfo;
    }

    @Override // com.lumiyaviewer.lumiya.voice.common.VoicePluginMessage
    public Bundle toBundle() {
        Bundle bundle = new Bundle();
        bundle.putBundle("channelInfo", this.channelInfo.toBundle());
        return bundle;
    }
}

File: app/src/main/java/com/lumiyaviewer/lumiya/voice/common/messages/VoiceChannelClosed.kt
package com.lumiyaviewer.lumiya.voice.common.messages

import android.os.Bundle
import com.lumiyaviewer.lumiya.voice.common.VoicePluginMessage
import com.lumiyaviewer.lumiya.voice.common.model.VoiceChannelInfo

class VoiceChannelClosed : VoicePluginMessage {
    @JvmField
    val channelInfo: VoiceChannelInfo

    constructor(bundle: Bundle) {
        this.channelInfo = VoiceChannelInfo(bundle.getBundle("channelInfo"))
    }

    constructor(voiceChannelInfo: VoiceChannelInfo) {
        this.channelInfo = voiceChannelInfo
    }

    override fun toBundle(): Bundle = Bundle().apply {
        putBundle("channelInfo", channelInfo.toBundle())
    }
}
